test/losses/test_hd.py

```diff
@@ -46,8 +45,9 @@ def perform_erosion(self, pred: np.ndarray, target: np.ndarray) -> np.ndarray:
         for batch in range(len(bound)):
             for k in range(self.erosions):
                 # compute convolution with kernel
-                dilation = convolve(bound[batch], kernel, mode="constant", cval=0.0)
-
+                dilation = torch.conv2d(
+                    torch.tensor(bound[batch], dtype=torch.float32), torch.tensor(kernel, dtype=torch.float32)
+                )[0].numpy()
```

Comment:
looks like this didn't work -- i think we can hardcode expected values instead of computing it on a numpy/scipy version (it's just used on `test_numeric`)

Reply:
+1